pandas/core/dtypes/dtypes.py

```diff
@@ -1,13 +1,15 @@
 """ define extension dtypes """
 import re
-from typing import Any, Dict, Optional, Tuple, Type
+import typing
```

Comment:
The preferred idiom is to just use `from typing import ...` so can add cast to the below line (if needed)

Reply:
Ok. Not even sure using ``cast`` is correct, it feels ugly.

Reply:
remove the typing, just import as indicated